locales/en/openbridge.json

```diff
@@ -46,9 +46,12 @@
   "instance": {
     "createASEInstance": "Create a Smart Events instance",
     "createSEInstance": "Create Smart Events instance",
+    "delete": "Delete instance",
+    "deleteIt": "Delete Smart Events Instance?",
```

Comment:
You use "...Smart Events **i**nstance..." elsewhere. Do you want to be consistent?